spec/validators/outgoing_incoming_trusts_ukprn_validator_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe OutgoingIncomingTrustsUkprnValidator do
+  subject { OutgoingIncomingTrustsUkprnValidatorTest.new }
+
+  context "when the outgoing and incoming trust UKPRNs are the same" do
```

Comment:
The contexts on here are the wrong way round - this one should be "when they are different"